.gitignore

```diff
@@ -0,0 +1,2 @@
+tasks.json
```

Comment:
This is not required, but I preferred to put some common patterns like *.json to a global ignore file to not repeat them in each repo. kinda like moving common code into a function. If you would need to track some specific json file, you can add it as an exception to the local repo with ! in the beginning of the rule.

my ignorefile is here https://github.com/nikandfor/configs/blob/master/home/nik/.gitignore